web/src/components/pages/data/data.tsx

```diff
@@ -224,9 +224,20 @@ class DataPage extends React.Component<Props, State> {
                 </div>
 
                 <div className="action">
-                  <CardAction onClick={() => this.showModalFor(dataset)}>
-                    <DownloadIcon />Download
-                  </CardAction>
+                  {dataset.name === 'LibriSpeech' ? (
```

Comment:
We could replace the LibriSpeech dataset-object's download property with an empty array, and check for that here.